contracts/rollup/Parameters.sol

```diff
@@ -1,9 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.6.12;
-
-contract Parameters {
-    uint256 public paramStakeAmount = 0.1 ether;
-    uint256 public paramBlocksToFinalise = 0;
-    uint256 public paramMinGasLeft = 10000;
-    uint256 public paramMaxTxsPerCommit = 32;
-}
```

Comment:
I move paramX to the place they are consumed.

Benefits:
- 1 less contract.
- They were visible where they are consumed.

Downsides:
- no centralized place to show them together. 

Note:
It's fine if we consume an immutable defined in the parent contract. No SLOAD is compiled in the child contract. But Solidity still required the function's visibility to be defined as "view", not "pure".


```
// SPDX-License-Identifier: MIT
pragma solidity ^0.6.12;

contract Parent {
    uint256 immutable a;
    constructor(uint256 _a) public{
        a = _a;
    }
    function viewParent() external view returns(uint256){
        return a;  // No SLOAD
    }
}
contract Child is Parent{
    constructor(uint256 _a) Parent(_a) public{}
    
    function viewChild() external view returns(uint256){
        return a; // No SLOAD
    }
}

```